### PR TITLE
Feature/auto create topics enable

### PR DESCRIPTION
**Pull request by dohq**
control `auto.create.topics.enable` properties

### CODE REVIEW

File: jobs/kafka/spec [hunk 1]
```diff
@@ -53,6 +53,10 @@ properties:
         replication_factor: 1
         partitions: 1
 
+  auto.create.topics.enable:
+    description: "Controls when Kafka enables auto creation of topic on the server."
+    default: true
```
Comment (drnic) [inline jobs/kafka/spec]: This changes the default behaviour of the bosh release. Could we start with just allowing the setting, and consider changing the default in future?

File: jobs/kafka/spec [hunk 2]
```diff
@@ -70,7 +74,7 @@ properties:
     default: 2
 
   advertised.listener:
-    desctiption: |-
+    description: |-
```
Comment (drnic) [inline jobs/kafka/spec:76]: Thanks.

**Review by drnic — CHANGES_REQUESTED** [review]
Bonus request: can you update `ci/release_notes.md` (or create it if its missing after a new release), with appropriate release notes? This allows me to quickly push your commits thru CI into new final releases.

See https://ci2.starkandwayne.com/teams/cfcommunity/pipelines/kafka-boshrelease for the pipeline

File: jobs/kafka/spec (hunk 1 above)
Comment (dohq) [reply jobs/kafka/spec]: I removed default section.
and rewrite `servers.properties`
please confirm it 👍

Comment (dohq) [reply jobs/kafka/spec]: And i wrote release notes.

**Comment by drnic** [comment]
I'm going to merge the PR; and manually fix the suggestions; then I can cut a new release for you.

**Pull request merged**